frontend/src/component/changeRequest/ChangeRequest/Changes/Change/FeatureChange.tsx

```diff
@@ -134,7 +134,14 @@ export const FeatureChange: FC<{
                 }
             />
 
-            <Box sx={(theme) => ({ padding: theme.spacing(3) })}>
+            <Box
```

Comment:
Should avoid SX when we can and use styled components, especially for complex styles that clutter the render function.

Reply:
Happy to switch to that, yeah!

Reply:
Addressed in e7807eed1fe76524ce6eb00b56170bce58c92f3e.

I know we've said not to use `sx`, but I still see it all over the code base. Is that an artefact from before we switched to styled components or is it acceptable in some cases?